language-support/ts/codegen/tests/ts/generated/src/__tests__/test.ts

```diff
@@ -72,17 +72,29 @@ afterAll(() => {
   console.log('Killed JSON API');
 });
 
-test('create + fetch & exercise', async () => {
-  const aliceLedger = new Ledger({token: ALICE_TOKEN, httpBaseUrl: HTTP_BASE_URL});
-  const bobLedger = new Ledger({token: BOB_TOKEN, httpBaseUrl: HTTP_BASE_URL});
-  const aliceStream = aliceLedger.streamQuery(Main.Person, {party: ALICE_PARTY});
-  const aliceIterator = pEvent.iterator(aliceStream, 'change', {rejectionEvents: ['close'], multiArgs: true});
-  const aliceIteratorNext = async () => {
-    const {done, value} = await aliceIterator.next();
+interface PromisifiedStream<T extends object, K, I extends string, State> {
+  next(): Promise<[State, readonly Event<T, K, I>[]]>;
+  close(): void;
+}
+
+function promisifyStream<T extends object, K, I extends string, State>(
+  stream: Stream<T, K, I, State>,
+): PromisifiedStream<T, K, I, State> {
+  const iterator = pEvent.iterator(stream, 'change', {rejectionEvents: ['close'], multiArgs: true});
+  const next = async () => {
+    const {done, value} = await iterator.next();
     expect(done).toBe(false);
```

Comment:
Why do we expect done to be false?

Reply:
Because the stream is infinite. We'll never be done with it.

Reply:
Ah right. 👍